### PR TITLE
FrontEnd Changes

### PR DESCRIPTION
**Pull request by mukkachaitanya**
Added much required changes for the frontend 😄.

### CODE REVIEW

File: templates/team.html [hunk 1]
```diff
@@ -423,7 +423,7 @@ <h3 style="margin-left:30%;margin-bottom: 2cm">Development Team</h3>
 
                                                 <p class="contacts">
                                                   <image src = "{% static 'img/icons/phone.png' %}" height="20">9590138612<br>
-                                                  <span class="fui-mail"></span> <a href="mailto: sebastin@bit-waves.org">mukka@bit-waves.org</a><br>
+                                                  <span class="fui-mail"></span> <a href="mailto: sebastin@bits-waves.org">mukka@bits-waves.org</a><br>
```
Comment (SebastinSanty) [inline templates/team.html]: mailto: mukka@bits-waves.org

**Comment by mukkachaitanya** [comment]
Couldn't squash the commits hence have two different commits.

**Review by SebastinSanty — COMMENTED** [review]
I have found that that the pre-waves button (on the top-left corner) cannot be clicked and also that the modal which opens up can't be closed properly.
The other things are working perfectly fine, and the logo is also centred. Can you consider making these changes too?

File: templates/team.html (hunk 1 above)
Comment (mukkachaitanya) [inline templates/team.html]: Did I leave that o_O !

**Comment by mukkachaitanya** [comment]
Squashed my commits. 😄

**Comment by SebastinSanty** [comment]
Can you create a new PR. The PreWaves has been removed.

**Pull request closed without merging**